salt/fileserver/gitfs.py

```diff
@@ -50,7 +50,7 @@
 
 PER_REMOTE_OVERRIDES = ('base', 'mountpoint', 'root', 'ssl_verify',
                         'env_whitelist', 'env_blacklist', 'refspecs')
-PER_REMOTE_ONLY = ('name', 'saltenv')
+PER_REMOTE_ONLY = ('formula', 'name', 'saltenv')
```

Comment:
Yeah. If you're comfortable writing RST, this should be documented in the gitfs documentation page at doc/topics/tutorials/gitfs.rst. If not, I can take care of documentation.